manifests/i/Insomnia/Insomnia/8.1.0/Insomnia.Insomnia.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.LF.7-3-7.Unix
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Insomnia.Insomnia
+PackageVersion: 8.1.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```
Is this a 64-bit application?